PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -1732,9 +1732,21 @@ intptr_t CALLBACK MarginsBorderEdgeSubDlg::run_dlgProc(UINT message, WPARAM wPar
 					return TRUE;
 
 				case IDC_CHECK_CHANGHISTORYMARGE:
-					svp._isChangeHistoryEnabled = (BST_CHECKED == ::SendDlgItemMessage(_hSelf, IDC_CHECK_CHANGHISTORYMARGE, BM_GETCHECK, 0, 0));
-					::SendMessage(::GetParent(_hParent), NPPM_INTERNAL_ENABLECHANGEHISTORY, 0, 0);
+				{
+					bool isChangeHistoryEnabled = (BST_CHECKED == ::SendDlgItemMessage(_hSelf, IDC_CHECK_CHANGHISTORYMARGE, BM_GETCHECK, 0, 0));
+					if (isChangeHistoryEnabled)
+					{
+						printStr(L"You have to restart Notepad++ to enable Change History.");

```

Comment:
@donho Is's worth adding a translation for this message.

Reply:
Sure. It's just a draft. I will make the whole PR commitable once everyone is on the same page.